src/main/java/com/nike/cerberus/command/StackDelegate.java

```diff
@@ -68,4 +70,25 @@ public CloudFormationParametersDelegate getCloudFormationParametersDelegate() {
     public Map<String, String> getDynamicParameters() {
         return dynamicParameters;
     }
+
+    public String[] getArgs(){
```

Comment:
This public method can use some javadocs. I can work on this.

Reply:
But it's going to be mostly just a copy of the same method in CreateInstanceProfileOperation.java